.github/workflows/linter.yml

```diff
@@ -15,7 +15,32 @@ jobs:
 
       - name: Check Docs
         run: |
-          docker run --rm --volume "$(pwd):/helm-docs" -u "$(id -u)" jnorwood/helm-docs:v1.4.0
+          OLD_CHARTS="./charts/mimir-distributed\n
+          ./charts/loki-distributed\n
+          ./charts/enterprise-logs\n
+          ./charts/tempo-distributed\n
+          ./charts/loki-canary\n
+          ./charts/grafana\n
+          ./charts/enterprise-logs-simple\n
+          ./charts/promtail\n
+          ./charts/fluent-bit\n
+          ./charts/enterprise-metrics\n
+          ./charts/loki\n
+          ./charts/agent-operator\n
+          ./charts/loki-stack\n
+          ./charts/tempo\n
+          ./charts/tempo-vulture\n
+          ./charts/loki-simple-scalable\n"
```

Comment:
Why we need this? Doesn't add charts to [.helmdocsignore](https://github.com/grafana/helm-charts/blob/main/.helmdocsignore) would help?

Reply:
nope, I want to introduce helm-docs v1.8.1 so people don't have to have an old version around, but simply switching means that all charts need to be modified, but we can only modify one chart at a time, so we need some mechanism to gradually change the helm-docs version. So far I couldn't fix it, I'll try again tomorrow my time (UTC+2)

Reply:
oh, wait I think I understand what you mean - that seems like a pretty good idea if it works :) going to try tomorrow, or feel free to try it yourself